src/app/common/hooks/use-swap-actions.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react';
+
+import { useAppDispatch } from '@app/store';
+import { swapsActions } from '@app/store/swap';
+
+import { MagicInboundSwap, MagicInboundSwapStatus } from '../magic/models';
+
+export function useSwapActions() {
```

Comment:
I don't know if I feel like this should be located here as a common hook? Should it just be co-located with swap hooks?

Reply:
I put it here because other actions hooks were also here, but yeah, it makes sense too to move it to the swap hooks folder to keep everything together.